### PR TITLE
Rotating color

### PR DESCRIPTION
**Pull request by stephalee**
## Changes

- Adds `RotatingColor` function and applies to links

## Steps To Test

- Go to an entry page and see link color slowly rotating!

### CODE REVIEW

**Comment by alecmarcus** [comment]
This is great!

Fundamentally, everything looks good. I think it would be useful to manage the RGB values and the final color as a CSS variables on the `:root` element though, which would give a few benefits:

1. We can manage the colors in CSS
1. We can use the dynamic color wherever we like, not just on links
1. We can apply the dynamic color to any property, not just `color`
1. We won't need to use JS to query for link elements on page load

I'll leave some comments on how you might do that!

File: src/assets/scripts/RotatingColor.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+class RotatingColor {
+  constructor() {
+    this.startingRed = 0;
```
Comment (alecmarcus) [inline src/assets/scripts/RotatingColor.js:3]: Instead, in `src/assets/styles/base/_global.scss`, we can add:

```scss
:root {
  ...
  --rotatingRed: 0;
  --rotatingGreen: 0;
  --rotatingBlue: 0;
  --rotatingColor: rgb(var(--rotatingRed), var(--rotatingGreen), var(--rotatingBlue));
  ...
}

**Review by bchiang7 — APPROVED** [review]
Very cool! 🌈 

Only suggestion I might have is preventing the color from getting too close to the regular text color (black) or else people might not recognize them as links if scrolling too quick)

**Comment by alecmarcus** [comment]
@bchiang7 thanks for the suggestion! it led me to do a small refactor, could you take another look?

**Comment by stephalee** [comment]
🌈 Merge away!

**Comment by bchiang7** [comment]
> @bchiang7 thanks for the suggestion! it led me to do a small refactor, could you take another look?

:shipit:

**Pull request merged**